keyboards/pistachio/info.json

```
@@ -0,0 +1,103 @@
+            {"label":"F2", "x":3.75, "y":0.25}, 
+            {"label":"F3", "x":4.75, "y":0.25}, 
+            {"label":"F4", "x":5.75, "y":0.25}, 
+            {"label":"DEL", "x":7.25, "y":0.25}, 
+            {"label":"F5", "x":9.5, "y":0.25}, 
+            {"label":"F6", "x":10.5, "y":0.25}, 
+            {"label":"F7", "x":11.5, "y":0.25}, 
+            {"label":"F8", "x":12.5, "y":0.25}, 
+            {"label":"F9", "x":14, "y":0.25}, 
+            {"label":"F10", "x":15, "y":0.25}, 
+            {"label":"F11", "x":16, "y":0.25}, 
+            {"label":"F12", "x":17, "y":0.25}, 
+            {"label":"\u25b6\u25b6|", "x":0, "y":1.75}, 
+            {"label":"E/J", "x":1.25, "y":1.75}, 
+            {"label":"!", "x":2.25, "y":1.75}, 
+            {"label":"\"", "x":3.25, "y":1.75}, 
+            {"label":"#", "x":4.25, "y":1.75}, 
+            {"label":"$", "x":5.25, "y":1.75}, 
+            {"label":"%", "x":6.25, "y":1.75}, 
+            {"label":"&", "x":7.25, "y":1.75}, 
+            {"label":"'", "x":10, "y":1.75}, 
+            {"label":"(", "x":11, "y":1.75}, 
+            {"label":")", "x":12, "y":1.75}, 
+            {"label":"", "x":13, "y":1.75}, 
+            {"label":"=", "x":14, "y":1.75}, 
+            {"label":"~", "x":15, "y":1.75}, 
+            {"label":"\\", "x":16, "y":1.75}, 
+            {"label":"\u2190", "x":17, "y":1.75}, 
+            {"label":"Delete", "x":18.25, "y":1.75}, 
+            {"label":"\u25b6||", "x":0, "y":2.75}, 
+            {"label":"Tab", "x":1.25, "y":2.75, "w":1.5}, 
+            {"label":"Q", "x":2.75, "y":2.75}, 
+            {"label":"W", "x":3.75, "y":2.75}, 
+            {"label":"E", "x":4.75, "y":2.75}, 
+            {"label":"R", "x":5.75, "y":2.75}, 
+            {"label":"T", "x":6.75, "y":2.75}, 
+            {"label":"Y", "x":9.5, "y":2.75}, 
+            {"label":"U", "x":10.5, "y":2.75}, 
+            {"label":"I", "x":11.5, "y":2.75}, 
+            {"label":"O", "x":12.5, "y":2.75}, 
+            {"label":"P", "x":13.5, "y":2.75}, 
+            {"label":"`", "x":14.5, "y":2.75}, 
+            {"label":"{", "x":15.5, "y":2.75}, 
+            {"label":"Enter", "x":16.75, "y":2.75, "w":1.25, "h":2}, 
+            {"label":"Home", "x":18.25, "y":2.75}, 
+            {"label":"\ud83d\udd07", "x":0, "y":3.75}, 
+            {"label":"CapsLock", "x":1.25, "y":3.75, "w":1.75}, 
+            {"label":"A", "x":3, "y":3.75}, 
+            {"label":"S", "x":4, "y":3.75}, 
+            {"label":"D", "x":5, "y":3.75}, 
+            {"label":"F", "x":6, "y":3.75}, 
+            {"label":"G", "x":7, "y":3.75}, 
+            {"label":"H", "x":9.75, "y":3.75}, 
+            {"label":"J", "x":10.75, "y":3.75}, 
+            {"label":"K", "x":11.75, "y":3.75}, 
+            {"label":"L", "x":12.75, "y":3.75}, 
+            {"label":"+", "x":13.75, "y":3.75}, 
+            {"label":"*", "x":14.75, "y":3.75}, 
+            {"label":"}", "x":15.75, "y":3.75}, 
```

Moving the Enter key due to the sequence change in the layout macro (`pistachio.h`).

```suggestion
            {"label":"Home", "x":18.25, "y":2.75}, 
            {"label":"\ud83d\udd07", "x":0, "y":3.75}, 
            {"label":"CapsLock", "x":1.25, "y":3.75, "w":1.75}, 
            {"label":"A", "x":3, "y":3.75}, 
            {"label":"S", "x":4, "y":3.75}, 
            {"label":"D", "x":5, "y":3.75}, 
            {"label":"F", "x":6, "y":3.75}, 
            {"label":"G", "x":7, "y":3.75}, 
            {"label":"H", "x":9.75, "y":3.75}, 
            {"label":"J", "x":10.75, "y":3.75}, 
            {"label":"K", "x":11.75, "y":3.75}, 
            {"label":"L", "x":12.75, "y":3.75}, 
            {"label":"+", "x":13.75, "y":3.75}, 
            {"label":"*", "x":14.75, "y":3.75}, 
            {"label":"}", "x":15.75, "y":3.75}, 
            {"label":"Enter", "x":16.75, "y":2.75, "w":1.25, "h":2}, 
```